ush/load_ufsda_modules.sh

```diff
@@ -27,48 +27,25 @@ fi
 ulimit_s=$( ulimit -S -s )
 
 # Find module command and purge:
+source "${HOMEgfs}/ush/detect_machine.sh"
 source "${HOMEgfs}/ush/module-setup.sh"
 
 # Load our modules:
 module use "${HOMEgfs}/sorc/gdas.cd/modulefiles"
 
-if [[ -d /lfs/f1 ]]; then
-  # We are on WCOSS2 (Cactus or Dogwood)
-  echo WARNING: UFSDA NOT SUPPORTED ON THIS PLATFORM
-elif [[ -d /lfs3 ]] ; then
-  # We are on NOAA Jet
-  echo WARNING: UFSDA NOT SUPPORTED ON THIS PLATFORM
-elif [[ -d /scratch1 ]] ; then
-  # We are on NOAA Hera
-  module load "${MODS}/hera"
-  # set NETCDF variable based on ncdump location
-  NETCDF=$( which ncdump )
-  export NETCDF
-elif [[ -d /work ]] ; then
-  # We are on MSU Orion or Hercules
-  if [[ -d /apps/other ]] ; then
-     # Hercules
-     module load "${MODS}/hercules"
-  else
-     # Orion
-     module load "${MODS}/orion"
-  fi
-  # set NETCDF variable based on ncdump location
-  ncdump=$( which ncdump )
-  NETCDF=$( echo "${ncdump}" | cut -d " " -f 3 )
-  export NETCDF
-elif [[ -d /glade ]] ; then
-  # We are on NCAR Yellowstone
-  echo WARNING: UFSDA NOT SUPPORTED ON THIS PLATFORM
-elif [[ -d /lustre && -d /ncrc ]] ; then
-  # We are on GAEA.
-  echo WARNING: UFSDA NOT SUPPORTED ON THIS PLATFORM
-elif [[ -d /data/prod ]] ; then
-  # We are on SSEC S4
-  echo WARNING: UFSDA NOT SUPPORTED ON THIS PLATFORM
-else
-  echo WARNING: UNKNOWN PLATFORM
-fi
+case "${MACHINE_ID}" in
+  ("hera" | "orion" | "hercules")
+    module load "${MODS}/${MACHINE_ID}"
+    NETCDF=$( command -v ncdump )
+    export NETCDF
```

Comment:
This will set `NETCDF` to the path of `ncdump`.
`command -v ncdump` yields `/apps/netcdf/4.7.0/intel/18.0.5.274/bin/ncdump`
We need `/apps/netcdf/4.7.0/intel/18.0.5.274` to be set as `NETCDF`

Reply:
Thank you, @aerorahul. Good catch. 

This should be resolved in the most recent push.